### PR TITLE
Run CodeFormatter on SvcUtil

### PR DESCRIPTION
**Pull request by shmao**

### CODE REVIEW

File: src/Common/src/System/SR.cs [hunk 1]
```diff
@@ -1,4 +1,7 @@
-// Copyright (c) Microsoft. All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
```
Comment (zhenlan) [inline src/Common/src/System/SR.cs]: Redundant line?

**Comment by zhenlan** [comment]
Can we add headers for tests as well?
https://github.com/dotnet/wcf/blob/master/src/svcutilcore/tests/SvcutilTests.cs
https://github.com/dotnet/wcf/blob/master/src/svcutilcore/tests/TestTypes.cs

**Comment by zhenlan** [comment]
@dotnet-bot test this please

File: src/Common/src/System/SR.cs (hunk 1 above)
Comment (shmao) [reply src/Common/src/System/SR.cs]: Yes, removed the line. Thanks.

**Comment by zhenlan** [comment]
Interesting. Not sure why CI is not kicking in. @StephenBonikowsky, any ideas?

**Comment by shmao** [comment]
@dotnet-bot test this please

**Pull request merged**